docs/docs/how-to-build-custom-client/how-to-build-custom-client.md

```diff
@@ -0,0 +1,481 @@
+---
+title: How to build custom client
+---
+
+We usually use [next.js](https://nextjs.org/) to build our site client. This guide will show you how to build a custom client for your Comet DXP and render a typical content page from the page tree.
+
+### Page Tree Node List
+
+To load/see all page tree node's, you need to query the `pageTreeNodeList` field in the `Query` type. This field allows you to fetch a list of `PageTreeNode` objects based on specific criteria such as `contentScope` and `category`. Those values differ from project to project and are defined by the project's requirements.
```

Comment:
We also have a paginatedPageTreeNodes query which is more performant -> should we recommend this one?

Reply:
i think especially in a tree it's hard to work with paginated stuff. I will add that there is also a more performant query, but i will not suggest one of those.